src/hooks/useDownload.ts

```diff
@@ -77,9 +84,11 @@ async function internal<T>({ src, dst, headers, ephemeral, logger }: DownloadOpt
       dst.parent().mkpath()
       await body(reader, f, sz)
 
-      //TODO etags too
       const text = rsp.headers.get("Last-Modified")
+      const etag = rsp.headers.get("etag")
```

Comment:
Case sensitivity doesn't matter, but we capitalized it like the docs do on the previous line so consistency would have been `ETag` per the docs for ETags.